http/README.md

```diff
@@ -33,6 +34,7 @@ services:
       MF_NATS_URL: [NATS instance URL]
       MF_HTTP_ADAPTER_LOG_LEVEL: [HTTP Adapter Log Level]
       MF_HTTP_ADAPTER_PORT: [Service HTTP port]
+      MF_CA_CERTS: [Path to trusted CAs in PEM format]
```

Comment:
Should this be `MF_HTTP_ADAPTER_CA_CERTS`?

Reply:
I think it should be `MF_THINGS_CA_CERTS`, as CA issues private/public cert pair for server - which in this case is `things`.